corehq/couchapps/export_instances_by_is_daily_saved/views/view/map.js

```diff
@@ -1,5 +1,5 @@
 function(doc) {
     if (doc.doc_type === "FormExportInstance" || doc.doc_type === "CaseExportInstance") {
```

Comment:
why not filter by `is_daily_saved_export` and `auto_rebuild_enabled` here instead of when querying the view?